balancer-visualization-prototype/ott-visualizationprototype-panel/package.json

```diff
@@ -0,0 +1,71 @@
+{
+  "name": "visualization-prototype",
```

Comment:
nit: Why'd you call it prototype? This is the permanent name of the package. Please make it something short, like `ott-vis`.